src/main/java/org/eclipse/pass/download/Serve.java

```diff
@@ -0,0 +1,27 @@
+package org.eclipse.pass.manuscript;
```

Comment:
Name the class something like DownloadServlet. Also make the "/lookup" to match the existing service and your documentation.